README.md

```diff
@@ -64,63 +64,6 @@ The Teleporter protocol, on the other hand, is implemented at the smart contract
   - `scripts/local/` includes scripts for running Teleporter in Docker.
 - `docker/` includes configurations for a local, containerized setup of Teleporter.
 
-## Run a local testnet in Docker
-
-A docker setup for running a local network with Teleporter deployed is provided. This setup provides a convenient way to develop and test Teleporter as well as cross-chain applications built on top of Teleporter. Teleporter messages are relayed between subnets using [AWM Relayer](https://github.com/ava-labs/awm-relayer), a fully featured implementation of a Warp message relayer.
-
-### Start up the local testnet
```

Comment:
Lets update this documentation to point to the Avalanche CLI as the easiest method for setting up local Teleporter enabled Subnets

Reply:
addressed in https://github.com/ava-labs/teleporter/pull/393/commits/3b96fdc691dcb716f740a8a6a2656ccd9f9bdf9e

i realize that's not _exactly_ what you asked for. :) (I pointed to the docs, which use the CLI, rather than directly to the CLI itself.) Let me know what you think.